### DISCUSSION TITLE
Error 429: Too Many Requests

### DISCUSSION THREAD

**Discussion by HomeMatic-INSIDE** [alexkn/node-red-contrib-homeconnect#42] [Q&A]
Hi,

since yesterday, the event has stopped working due of an API error 429 (too many requests):

`"{"type":"error","status":429,"message":"Too Many Requests"}"`

How to solve this?

Thanks,...

**Comment by alexkn** [comment]
It sometimes happens when you restart Node-Red too often.
Usually this should fix itself after a while.

**Reply by HomeMatic-INSIDE** [reply]
Not for me - I've waited two days until I've created a new Home Connect Application and reconnected it with the new credentials. Now, it work's again...but I'll keep an eye on it ;-) BTW: Thank you for this node ;-)
